WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationLocalStore.swift

```diff
@@ -398,10 +385,24 @@ public final class ConversationLocalStore: ConversationLocalStoreProtocol {
         _ participants: [(id: UUID, domain: String?, role: String?)],
         addedBy sender: (id: UUID, domain: String?),
         atDate date: Date,
-        to conversation: ZMConversation
+        conversation: (id: UUID, domain: String)
     ) async throws {
         typealias UserAndRole = (user: ZMUser, role: Role?)
 
+        let localConversation = await fetchConversation(
+            id: conversation.id,
+            domain: conversation.domain
+        )
+
+        guard let localConversation else {
+            return WireLogger.eventProcessing.error(
```

Comment:
I don't think it's a common pattern, I use it to have a single line but it might be a bit confusing, might be interesting to add this to the convention discussion. I'll add it to the document thx